xbmc/interfaces/json-rpc/schema/types.json

```diff
@@ -1016,7 +1016,8 @@
       "icon": { "type": "string" },
       "channelnumber": { "type": "integer" },
       "subchannelnumber": { "type": "integer" },
-      "isrecording": { "type": "boolean" }
+      "isrecording": { "type": "boolean" },
+      "hasarchive": { "type": "boolean" }
```

Comment:
Thanks for updating the API as part of this feature, and for correctly adding the field at the end of the type.

However addition of fields to the JSON schema requires a bump to the _minor_ version number (the middle one) in https://github.com/xbmc/xbmc/blob/master/xbmc/interfaces/json-rpc/schema/version.txt as a non-breaking change, communicating to API customers that things have changed. Adding [JSON] to the commit comment would also make it easier to spot.

Reply:
Thanks. I'll do that once I'm home.
Should I make another commit, or amend the previous one and force push? I'm not yet familiar with how PRs to Kodi work.

Reply:
You can amend and force push changes. You can just do Fixup commit and in the end squash them into clear separated changes.

Reply:
Done.